internal/cmd/configuration/command_set.go

```diff
@@ -0,0 +1,119 @@
+package configuration
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/form"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *command) newSetCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set <config-field-1=value-1> ... [config-field-n=value-n]",
```

Comment:
To match our other commands?
```suggestion
		Use:   "update <config-field-1=value-1> ... [config-field-n=value-n]",
```